build.sbt

```
@@ -16,20 +16,23 @@ lazy val baseSettings: Seq[Setting[_]] = Seq(
 lazy val intro = project.in(file("."))
   .settings(moduleName := "scala-intro")
   .settings(baseSettings: _*)
-  .aggregate(core, slides)
-  .dependsOn(core, slides)
+  .aggregate(core, docs)
+  .dependsOn(core, docs)
 
 lazy val core = project
   .settings(moduleName := "scala-intro-core")
   .settings(baseSettings: _*)
 
+lazy val docs = project.in(file("slides2"))
+  .dependsOn(core)
```

What happens if you do? I'm wondering if the scala version in the `docs` project is picking up the default 2.12.x value from sbt
```suggestion
  .settings(baseSettings)
  .dependsOn(core)
```